brainstormforce/astra
# AST-3314 - Fix: do not display banner on admin theme upload page.

### Description

- Admin Notice is breaking on Theme Upload Page.
- JIRA :- https://brainstormforce.atlassian.net/browse/AST-3314 

### Screenshots

https://d.pr/i/m57c5e

### Types of changes




Bug Fixes

### How has this been tested?

- Reset site or fresh setup install.
- Checkout to this branch.
- Install Any Latest Astra Zip or Download Zip from Github.
- The Admin Welcome Banner Notice should not be visible.

### Checklist:
